sonic-post-syseeprom/setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import setup
+
+setup(
+    name='sonic-post-syseeprom',
+    version='1.0',
+    description='Syseeprom gathering task for SONiC',
```

Comment:
Suggest change description to "Tool which posts syseeprom info to DB in SONiC"

Reply:
revised